lib/messages/messages.dart

```diff
@@ -34,7 +34,8 @@ class MessageId implements Comparable {
 
   toString() => '${package}#$id';
 
-  int compareTo(MessageId other) {
+  int compareTo(_other) {
```

Comment:
This is a bit of a weird one, is comparable a generic class? Ideally `MessageId` would extend `Comparable<MessageId>` and then you could leave the correct type in here.